### PR TITLE
Update release notes instructions in Contributing docs

### PR DESCRIPTION
**Pull request by dvejmz**
Make release notes instructions slightly more specific, which may be useful for first time contributors. News fragment update will follow soon.

### CODE REVIEW

File: news/PR238.other [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve release notes process documentation.
```
Comment (abompard) [inline news/PR238.other]: I would suggest the type `docs` instead of `other` here.

File: docs/contributing.rst [hunk 2]
```diff
@@ -57,8 +60,8 @@ you're not certain how to write tests, we will be happy to help you.
 
 Release notes
 -------------
-To add entries to the release notes, create a file in the ``news`` directory
-with the ``source.type`` name format, where ``type`` is one of:
+
+To add entries to the release notes, run ``towncrier create <source.type>`` to create a news fragment file in the ``news`` directory, where ``type`` is one of:
```
Comment (abompard) [inline docs/contributing.rst]: Could you wrap this line like the others please?

File: docs/contributing.rst [hunk 3]
```diff
@@ -77,6 +80,14 @@ And where the ``source`` part of the filename is:
 * ``username`` for contributors (``author`` extention). It should be the
   username part of their commits' email address.
 
+For example,
+
+::
+
+    towncrier create PR42.feature    
+
+The contents of the news fragment must be written in RST format. See the `towncrier documentation for more information <https://github.com/twisted/towncrier>`_.
```
Comment (abompard) [inline docs/contributing.rst]: Same here

**Review by abompard — CHANGES_REQUESTED** [review]
Thanks! I hadn't noticed that towncrier now has subcommands.

**Comment by dvejmz** [comment]
Thanks for your comments @abompard , I've now addressed them all.

File: docs/contributing.rst [hunk 4]
```diff
@@ -77,6 +82,16 @@ And where the ``source`` part of the filename is:
...
   username part of their commits' email address.
 
+For example,
+
+::
+
+    towncrier create PR42.feature    
+
+The contents of the news fragment must be written in RST format.
+See the `towncrier documentation formore information
```
Comment (abompard) [inline docs/contributing.rst]: Missing space here

**Review by abompard — CHANGES_REQUESTED** [review]
I'm only getting around to looking at this now, sorry. Just a small typo. Thanks!

**Comment by dvejmz** [comment]
No problem, I've now pushed a fix @abompard

**Review by abompard — APPROVED** [review]

**Pull request merged**